easystacks/software.eessi.io/2023.06/eessi-2023.06-eb-4.8.2-2023a.yml

```diff
@@ -17,3 +17,4 @@ easyconfigs:
       # see https://github.com/easybuilders/easybuild-easyconfigs/pull/19268
       options:
         from-pr: 19268
+  - Qt5-5.15.10-GCCcore-12.3.0.eb
```

Comment:
@Neves-P will need merge conflict resolution after #405 is merged (but you know that already, just making a note of it)